builder/processor/src/main/java/io/helidon/builder/processor/AnnotationDataOption.java

```diff
@@ -316,6 +327,40 @@ boolean hasAllowedValues() {
         return allowedValues != null && !allowedValues.isEmpty();
     }
 
+    private static Javadoc processDeprecation(DeprecationData deprecationData, List<Annotation> annotations, Javadoc javadoc) {
+        if (javadoc == null) {
+            return null;
+        }
+
+        if (!deprecationData.deprecated()) {
+            return javadoc;
+        }
+
+        io.helidon.common.types.Annotation.Builder deprecated = io.helidon.common.types.Annotation.builder()
+                .typeName(DEPRECATED_TYPE);
+        if (deprecationData.since() != null) {
+            deprecated.putValue("since", deprecationData.since());
+        }
+        if (deprecationData.forRemoval()) {
```

Comment:
replace with `deprecated.putValue("forRemoval", deprecationData.forRemoval())`

Reply:
No, because the annotation property should only be generated when it is set to `true`, as `false` is default